code/datums/shuttles.dm

```diff
@@ -88,14 +95,17 @@
 		crates!"
 	admin_notes = "Due to origin as a solo piloted secure vessel, has an \
 		active GPS onboard labeled STV5."
+	credit_cost = -7500
```

Comment:
So you can get a shittier shuttle in exchange for a short term benefit, it's a trade off